src/actions/UserActions.ts

```diff
@@ -37,3 +37,24 @@ export const handleUserLogin = ({ member_code, password }: userAuthData) => {
       });
   });
 };
+
+export const getSpecificUser = (id: string) => {
+  return new Promise<User>((resolve, reject) => {
+    useQuery<getUserReturnType>(USER_BY_ID, { id })
+      .then(({ users_by_pk: user }) => {
+        if (!user) {
+          Notify.create({
+            message: 'No se encontro el usuario',
+            type: 'negative',
+          });
+
+          reject(null);
+        }
```

Comment:
I remember that after of reject, the function continues, it's the reason because I returned after of reject, please check and confirm if this function works fine without return after the reject